packages/powershell/oh-my-posh/oh-my-posh.psm1

```diff
@@ -92,11 +92,11 @@ function Set-PoshPrompt {
         $standardOut = $process.StandardOutput.ReadToEnd()
         $process.WaitForExit()
         $standardOut
+        Set-GitStatus
```

Comment:
@JanDeDobbeleer I'm not actually sure what `Set-GitStatus` does, so I'm not sure what to test to see if this broke anything. I find it odd that `Set-GitStatus` was/is called AFTER executing the ohmyposh executable. Can you shed some insight on this?

Reply:
Yes. It makes sure to populate the `posh-git` status when that module is present. It does that by itself by setting the prompt but as we override that, that context (and the accompanying autocomplete) is broken.